package.json

```diff
@@ -23,14 +23,38 @@
     "build/*"
   ],
   "exports": {
-    ".": "./build/index.js",
-    "./plugins": "./build/index.plugins.js",
-    "./core": "./build/index.core.js",
-    "./three": "./build/index.three.js",
-    "./r3f": "./build/index.r3f.js",
-    "./core/plugins": "./build/index.core-plugins.js",
-    "./three/plugins": "./build/index.three-plugins.js",
-    "./src/*": "./src/*"
+    ".": {
+		"import": "./build/index.js",
+		"types": "./build/types/index.d.ts"
```

Comment:
Amazing, thanks! Can you explain why all the d.ts files need to be copied to the `/build` folder rather than be directly referenced in the `/src` folder?

Reply:
They don't. Both ways work; just a "design" choice. It was my initial proposal in the issue to keep consistency with your build folder approach to have there all the files exported by the library. This includes the types IMO. But you could also reference the files in the src folder and remove the copying step in the vite config file. Your choice :)